src/main/resources/swagger/thurloe.yaml

```diff
@@ -329,6 +329,9 @@ paths:
               "userId": {
                 "type": "string"
               },
+              "fromAddress": {
+                "type": "string"
+              },
```

Comment:
good Q. I tried it out and it didn't go to my spam folder, but the reply-to address shows up in gmail as:

`mbemisbroad@gmail.com via dev.test.firecloud.org`

SendGrid hints that it depends on the mail client- some recognize that the domain of the sender doesn't match what's in the DKIM signature so they add the "via" to prevent phishing, but I wouldn't be surprised if others handle it more strictly or leniently 

Honestly, I don't like this part of the story and think that we should use things like notifications@firecloud.org, help@firecloud.org, etc. We're already saying "mbemisbroad@gmail.com added you to the workspace foo/bar" in the email itself. It seems weird to be forging emails on behalf of a user